JWBestPracticeApps/Podfile

```diff
@@ -1,31 +1,35 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '8.0'
+# platform :ios, '11.0'
 # Uncomment this line if you're using Swift
 # use_frameworks!
 
+def common_JW
+    pod 'JWPlayer-SDK', '~> 3.2.0'
```

Comment:
wouldn't this limit us to only 3.2.x ? We will have to change this for 3.3.0 right ?
let's use `pod 'JWPlayer-SDK', '~> 3.0'`

Reply:
@karimMourra It's changed.